### PR TITLE
replace testsuite to testify part 1(fix #20)

### LINKED ISSUE #20: use /stretchr/testify package instead of own assert functions for testing
**Description**

I think it's not needed to reinvent the wheel when there is already a great and feature packed assert package that has everything that could be needed to test the codebase. Please checkout [testify](https://github.com/stretchr/testify) [(godoc)](https://godoc.org/github.com/stretchr/testify/assert).

**Steps to reproduce the behavior:**

**Expected behavior**

**OS and Other informations**

**Additional details**

### PR DESCRIPTION
**Pull request by Chyroc**
# Pull Request for issue

#20

## Description

* replace testsuite to testify
* install testify by dep

### CODE REVIEW

File: pkg/util/strs_test.go [hunk 1]
```diff
@@ -27,35 +27,37 @@ package util
 import (
 	"testing"
 
-	"github.com/kasvith/kache/pkg/testsuite"
+	"github.com/stretchr/testify/assert"
```
Comment (kasvith) [inline pkg/util/strs_test.go]: import this as testify like `testifyAssert github.com/testify/assert`

File: pkg/util/strs_test.go [hunk 2]
```diff
@@ -27,35 +27,37 @@ package util
...
 )
 
 func TestSplitSpacesWithQuotes(t *testing.T) {
 	// test for balanced quotes
+	as := assert.New(t)
```
Comment (kasvith) [inline pkg/util/strs_test.go]: Then we can use `assert := testifyAssert.New(t)` which makes code more readiable

**Comment by Chyroc** [comment]
I went to take a shower and fixed these problems.

**Comment by kasvith** [comment]
Tnx @Chyroc

**Pull request merged**